pontoon/base/models/user.py

```diff
@@ -216,6 +217,47 @@ def contributed_translations(self):
     return Translation.objects.filter(user=self)
 
 
+@property
+def badges_translations_count(self):
+    """Contributions provided by user that count towards their badges."""
+    from pontoon.base.models.translation import Translation
+
+    badges_inception = aware_datetime(2024, 10, 18)
+
+    return Translation.objects.filter(user=self, date__gte=badges_inception).count()
```

Comment:
Translations can be removed. We should use `ActionLog`:
https://github.com/mozilla/pontoon/pull/3413#discussion_r1805000532

Additionally, start day of the observation period should be defined in one place only, possibly in settings.